manifests/03-cluster-operator.yaml

```diff
@@ -1,7 +1,7 @@
+# This resource is not created by CVO, it is only used as a communication
+# mechanism to tell the CVO which ClusterOperator resource to wait for.
+# IngressController creates and updates this resource.
```

Comment:
Does this resource's existence in `/manifests` mean that the CVO does in fact create the resource?

Reply:
Nope:

> Also note that the ClusterOperator resource in `/manifests` is only a communication mechanism, to tell the ClusterVersionOperator, which ClusterOperator resource to wait for. The ClusterVersionOperator does not create the ClusterOperator resource, creating and updating it is the responsibility of the respective operator.

https://github.com/openshift/cluster-version-operator/blob/8402d219f36fc79e03edf45918785376113f2cc1/docs/dev/clusteroperator.md#how-clusterversionoperator-handles-clusteroperator-in-release-image